<?xml version='1.0' ?>
<launch>
  <node pkg="can_message_handler" exec="can_message_sender_node" name="can_message_sender">
    <param name="can_ids" value="0x001,0x002,0x003"/>
    <param name="is_extended" value="false"/>
    <param name="dlc" value="8"/>
    <param name="period_ms" value="10"/>
  </node>

  <node pkg="can_message_handler" exec="can_message_receiver_node" name="can_message_receiver">
  </node>
</launch>